core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceEditorDirective.js

```diff
@@ -41,6 +41,15 @@ oppia.directive('multipleChoiceEditor', [function() {
         // Note that a questionId generated in this way may contain spaces,
         // since it is just the state name.
         $scope.questionId = $scope.getUniqueId();
+
+        $scope.getHash = function(suffix) {
```

Comment:
I'm not sure what additional purpose this `getHash` does. As far as I can see it's used when setting the 4 id's in MultipleChoiceInput below. Is the `getHash` ever used to "find" the ID again? Or is the purpose just to guarantee unique id's?